lark/tree.py

```diff
@@ -18,6 +16,9 @@
     else:
         from typing_extensions import Literal
 
+###{standalone
+from collections import OrderedDict
```

Comment:
Wasn't the goal to move all imports into `standalone.py`?

Reply:
Not exactly, I was mostly concerned about repetitive imports.

OrderedDict is only used once (and it a little meaningless in Python3, I just didn't think removing it fits into this PR)

Reply:
Fair enough. (Also `OrderedDict` is still needed for `3.6` on non-CPython.)